src/coreclr/jit/block.cpp

```diff
@@ -186,6 +186,21 @@ FlowEdge* Compiler::BlockPredsWithEH(BasicBlock* blk)
     return res;
 }
 
+//------------------------------------------------------------------------
+// IsLastHotBlock: see if this is the last block before the cold section
```

Comment:
Should we have a corresponding `IsFirstColdBlock(Compiler*)`?

Reply:
I can add that in for consistency.